scu-es/src/fiskaltrust.Middleware.SCU.ES.TicketBAI/Models/TicketBaiRequest.cs

```diff
@@ -0,0 +1,1188 @@
+        Item14,
+
+        /// <summary>
+        /// Segidako traktuko eragiketetan ordaintzeke dagoen BEZdun faktura - Factura con IVA pendiente de devengo en operaciones de tracto sucesivo
+        /// </summary>
+        [XmlEnum("15")]
+        Item15,
+
+        /// <summary>
+        /// IX. tituluko XI. kapituluan aurreikusitako araubideren bati atxikitako eragiketa (OSS eta IOSS) - Operación acogida a alguno de los regímenes previstos en el Capítulo XI del Título IX (OSS e IOSS)
+        /// </summary>
+        [XmlEnum("17")]
+        Item17,
+
+        /// <summary>
+        /// Nekazaritza, abeltzaintza eta arrantzaren araubide berezian dauden jardueren eragiketak (NAAAB) - Operaciones de actividades incluidas en el Régimen Especial de Agricultura, Ganadería y Pesca (REAGYP)
+        /// </summary>
+        [XmlEnum("19")]
+        Item19,
+
+        /// <summary>
+        /// Baliokidetasun errekarguko eragiketak - Operaciones en recargo de equivalencia
+        /// </summary>
+        [XmlEnum("51")]
+        Item51,
+
+        /// <summary>
+        /// Erregimen erraztuko eragiketak - Operaciones en régimen simplificado
+        /// </summary>
+        [XmlEnum("52")]
+        Item52,
+
+        /// <summary>
+        /// BEZaren ondorioetarako enpresari edo profesionaltzat jotzen ez diren pertsona edo erakundeek egindako eragiketak - Operaciones realizadas por personas o entidades que no tengan la consideración de empresarios, empresarias o profesionales a efectos del IVA
+        /// </summary>
+        [XmlEnum("53")]
+        Item53
+    }
+
+    [Serializable]
+    [XmlType("TipoDesgloseType", Namespace = "urn:ticketbai:emision")]
+    public class TipoDesgloseType
+    {
+        [XmlElement("DesgloseFactura")]
+        public DesgloseFacturaType DesgloseFactura { get; set; }
+
+        [XmlElement("DesgloseTipoOperacion")]
+        public DesgloseTipoOperacionType DesgloseTipoOperacion { get; set; }
+    }
+
+    [Serializable]
+    [XmlType("DesgloseFacturaType", Namespace = "urn:ticketbai:emision")]
+    public class DesgloseFacturaType
+    {
+
+        [XmlElement("Sujeta")]
+        public SujetaType Sujeta { get; set; }
+
+        [XmlArray("NoSujeta")]
+        [XmlArrayItem("DetalleNoSujeta")]
+        public List<DetalleNoSujeta> NoSujeta { get; set; }
+    }
+
+    [Serializable]
+    [XmlType("SujetaType", Namespace = "urn:ticketbai:emision")]
+    public class SujetaType
+    {
+        [XmlArray("Exenta")]
+        [XmlArrayItem("DetalleExenta")]
+        public List<DetalleExentaType> Exenta { get; set; }
+
+        [XmlArray("NoExenta")]
+        [XmlArrayItem("DetalleNoExenta")]
+        public List<DetalleNoExentaType> NoExenta { get; set; }
+    }
+
+    [Serializable]
+    [XmlType("ExentaType", Namespace = "urn:ticketbai:emision")]
+    public class ExentaType
+    {
+        [Required]
+        [XmlElement("DetalleExenta")]
+        public List<DetalleExentaType> DetalleExenta { get; set; }
+    }
```

Comment:
It looks like it may be duplicated to lines 888-890 as the List<> is declared too.